src/igvfd/schemas/changelogs/tabular_file.md

```diff
@@ -20,6 +21,7 @@
 * Add `barcode to sample mapping` to `content_type`.
 * Add `release_timestamp`.
 * Add `MPRAbase` to `collections`.
+* Add `integrated_in`.
```

Comment:
why do you report add it here, but then again that it was added as calculated?

Reply:
fixed